sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ProactiveContainerInitConfig.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
```

Comment:
Should be in models package?

Reply:
I am not too sure about this, other config classes such as `GlobalThroughputConfig` and `ThroughputControlConfig` are in the outside the models and the implementation packages.

Reply:
Discussed - leaving it here for now - will touch base on in Azure Central API review